### DISCUSSION TITLE
是否支持IP V6？是否能能够下载阿里云盘文件到本地？

### DISCUSSION THREAD

**Discussion by zcy85611** [messense/aliyundrive-webdav#377] [Q&A] [answered]
使用的openwrt路由器，mips架构，查看了服务监听的端口，仅监听了ip v4的端口，我也尝试过ip v6连接，防火墙端口已打开但是无法成功（证书，域名都没问题，自己建的webdav服务器ipv6连接正常）

另外是否有把阿里云盘文件下载到本地指定位置的功能？就像baidupcs的那种功能

有知道的朋友请帮助解答一下，谢谢

**Comment by messense** [comment, marked as answer]
学会搜索：https://github.com/messense/aliyundrive-webdav/discussions?discussions_q=ipv6

> 另外是否有把阿里云盘文件下载到本地指定位置的功能？就像baidupcs的那种功能

否，也不会支持。
